.github/issue_template.md

```diff
@@ -1,8 +1,10 @@
-[Describe your issue]
+#### Describe your issue
+
+...
 
 #### Check all that apply (change to `[x]`)
 - [ ] Windows
 - [ ] Mac OS X
 - [ ] Linux
 
-See <https://libigl.github.io/CONTRIBUTING/#bugreport> for more tips.
+See https://libigl.github.io/CONTRIBUTING/#bugreport for more tips.
```

Comment:
why these changes?

Reply:
The branch probably duplicates an old commit. The new version after merged is unchanged.

Reply:
@teseoch can we merge this?